RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -1344,6 +1313,15 @@ boolean L11_unlockHiddenCity()
 	auto_log_info("Searching for the Hidden City", "blue");
 	if(!in_glover() && !in_tcrs()) 
 	{
+		if(item_amount($item[Stone Wool]) == 0 && have_effect($effect[Stone-Faced]) == 0)
+		{
+			//attempt to summon before using a clover
+			handleFamiliar("item");
```

Comment:
similar to ghost a maximizer call for item 400max should go here?

Reply:
Good call, did it